src/ipc/Port.cc

```diff
@@ -79,6 +79,7 @@ void Ipc::Port::noteRead(const CommIoCbParams& params)
            " [" << this << ']');
     if (params.flag == Comm::OK) {
         assert(params.buf == buf.raw());
+        debugs(54, 6, "received message type: " << buf.rawType());
```

Comment:
Nitpick: "received" is implied by "noteRead" so we can make this shorter/simpler.

```suggestion
        debugs(54, 6, "message type: " << buf.rawType());
```

Reply:
Done.